contrib/jenkins_tests/globals.sh

```diff
@@ -6,6 +6,14 @@ WORKSPACE=${WORKSPACE:=$(pwd)}
 BUILD_NUMBER=${BUILD_NUMBER:=0}
 HOSTNAME=${HOSTNAME:=$(uname -n 2>/dev/null)}
 
+# mounting hugetlbfs for Gtest on xlio_doca: HPCINFRA-1968
+if [[ -f /.dockerenv && ! $(mount -t hugetlbfs nodev /mnt/huge) ]]; then
```

Comment:
and since its related to gtest it should not be in this commit which is related to compilation